locations/spiders/spotlight.py

```
@@ -1,70 +1,48 @@
-
-    def parse_token_request(self, response):
-        token_request_url = str(response.headers.getlist("Location")[0].decode("utf-8"))
-        yield scrapy.Request(url=token_request_url, callback=self.parse_token)
-
-    def parse_token(self, response):
-        url = str(response.headers.getlist("Location")[0].decode("utf-8"))
-        token = str(response.headers.getlist("Set-Cookie")[0].decode("utf-8"))
-        yield scrapy.Request(url=url, headers={"cookie": token}, callback=self.parse_store_urls)
+    def parse(self, response: Response) -> Iterable[Feature]:
+        store_json = response.xpath('//script[@id="storeJson"]/text()').get()
+        ld = chompjs.parse_js_object(
```

Flake8 (7.3.0)

[error] local variable 'ld' is assigned to but never used

(F841)